src/AccessibilityInsights/Misc/CommandOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using CommandLine;
+
+namespace AccessibilityInsights.Misc
+{
+    public class CommandOptions
+    {
+        [Value(0, HelpText = "File to open in Accessibility Insights", Required = false)]
```

Comment:
>HelpText [](start = 18, length = 8)

I'm not familiar with this framework. Where does the help text come into play?

Reply:
@DaveTryon For normal console apps running `program --help` should generate some help text. It looks like that's not working for the WPF app, so I can remove it & replace with a // comment. We could also just leave it in for the same documentation purpose. Do you prefer one or the other?

Reply:
I'd suggest removing the HelpText fields

---
In reply to: [298774055](https://github.com/microsoft/accessibility-insights-windows/pull/407#discussion_r298774055) [](ancestors = 298774055)

Reply:
+1 to removing these, at least the new custom config ones, since those aren't really meant for typical users.